XenAdmin/XenAdmin.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>WinExe</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>XenAdmin</RootNamespace>
-    <AssemblyName>XenCenterMain</AssemblyName>
+    <AssemblyName>CitrixHypervisorManagementMain</AssemblyName>

```

Comment:
`[XenCenter]Main` (I think it should compile); then rebrand this file with rebranding_global in the build scripts.

Reply:
I've tried to use [XenCenter_No_Space]Main here, but this is not replaced with "CitrixHypervisorManagementMain" during the build. ([XenCenter_No_Space] in other files could be replaced). So I think we have to hardcode here.

Reply:
Check whether Xenadmin.csproj is run through rebranding_global in the rebranding.sh.

Reply:
ok, it works now!